internal/webhook/module.go

```diff
@@ -207,3 +212,14 @@ func validateModprobe(modprobe kmmv1beta1.ModprobeSpec) error {
 
 	return nil
 }
+func validateModuleTolerarations(mod *kmmv1beta1.Module) error {
+	for _, toleration := range mod.Spec.Tolerations {
+		switch toleration.Effect {
+		case corev1.TaintEffectNoSchedule, corev1.TaintEffectNoExecute, corev1.TaintEffectPreferNoSchedule:
+			continue
+		default:
+			return fmt.Errorf("invalid toleration effect: %s. allowed values are NoSchedule, NoExecute, PreferNoSchedule", toleration.Effect)
```

Comment:
```suggestion
			return fmt.Errorf("invalid toleration effect %s. allowed values are NoSchedule, NoExecute, PreferNoSchedule", toleration.Effect)
```

We use `:` to wrap errors, having `:` in the string error will let us think that everything before the `:` is the wrapping error and everything after is the wrapped error.